mdx-models/src/main/java/com/mx/path/model/mdx/model/authorization/Authorization.java

```diff
@@ -11,6 +11,7 @@ public class Authorization extends MdxBase<Authorization> {
   private NameValuePair[] headers;
   private JavaScript javascript;
   private String token;
+  private NameValuePair[] tokens;
```

Comment:
Just curious what these token are used for which cannot be covered in headers or URL?

Reply:
I need to be able to return 2 tokens to the frontend (idToken and accessToken). We currently only have one token field which wont work for my situation.

Reply:
Also accessToken was never meant to be given to mobile or other third party service. It can cause some security concerns. Only thing that mobile can/should have access to is IdToken and Refresh Token.

Reply:
But they're not headers. They're tokens used for authentication. The Glia documentation specifically asks for these tokens

Reply:
Are these separate tokens from the ForgeRock idToken/accessToken?

I think the intention here is to avoid having to populate headers regularly when we can support any number of tokens needed long term for webviews. This avoids needing to make any changes in Synchronicity, only needing changes in connectors and helios code.

Reply:
Had a call with Tess and went through the flow. Understood why this is needed. i am fine with this.